### ISSUE TITLE
Command palette not opening

### ISSUE THREAD

**Issue by akissinger** [fman-community/fman#400]
Command palette refuses to open, and gives error below. OS is windows 10. Closing and re-opening fman fixed the issue. I have one plugin installed (Sublime text), otherwise installation is vanilla.

```
Command 'CommandPalette' raised error.

Traceback (most recent call last):
  File "fman\impl\plugins\command_registry.py", line 97, in _execute_command
  File "C:\Users\aleks\AppData\Local\fman\Versions\1.0.9\Plugins\Core\core\commands\__init__.py", line 1231, in __call__
    for quicksearch_item in self._suggest_commands(self._last_query)
  File "C:\Users\aleks\AppData\Local\fman\Versions\1.0.9\Plugins\Core\core\commands\__init__.py", line 1252, in _suggest_commands
    for cmd_name, aliases, command in self._get_all_commands():
  File "C:\Users\aleks\AppData\Local\fman\Versions\1.0.9\Plugins\Core\core\commands\__init__.py", line 1273, in _get_all_commands
    if not self.pane.is_command_visible(cmd_name):
  File "fman\__init__.py", line 83, in is_command_visible
  File "fman\impl\plugins\command_registry.py", line 89, in is_command_visible
  File "C:\Users\aleks\AppData\Local\fman\Versions\1.0.9\Plugins\Core\core\commands\__init__.py", line 774, in is_visible
    return bool(clipboard.get_files())
  File "fman\impl\util\qt\thread.py", line 11, in result
  File "fman\impl\util\qt\thread.py", line 58, in run_in_thread
  File "fman\impl\util\qt\thread.py", line 83, in result
  File "fman\impl\util\qt\thread.py", line 71, in __call__
  File "fman\clipboard.py", line 53, in get_files
  File "fman\clipboard.py", line 53, in <listcomp>
  File "fman\impl\util\qt\__init__.py", line 55, in from_qurl
ValueError: not enough values to unpack (expected 2, got 1)
```

**Comment by mherrmann** [comment]
Thank you for letting me know Aleks (and nice to hear from you again). I'll take a look.

**Comment by mherrmann** [comment]
It seems there was some weird file (or other element) on your clipboard. Any chance you can remember what it was?

**Comment by JRosanowski** [comment]
I got the same problem. I had this on my clipboard.

if (isSyncing) {
    isSyncing = false
    startPhotoUploader()
}

**Comment by mherrmann** [comment]
Thank you @JRosanowski. Unfortunately, I can't reproduce the problem even when I have what you pasted on the clipboard. What were you doing at the time the problem occurred? Did you maybe also "copy" something else? Like (say) a file in your native file manager. Or an element in MS Office. Or an image. Or ...?

**Comment by mherrmann** [comment]
I believe I was able to fix this with a change in [fman 1.1.1](https://fman.io/changelog#v1.1.1).
